server/handlers/user.go

```diff
@@ -40,16 +40,15 @@ func registerUser(w http.ResponseWriter, res *http.Request) {
 	token.SendJWT(w, jwtToken)
```

Comment:
Before this line you can write a status code of created to the `ResponseWriter`. You can use `ResponseWriter.WriteHeader` for that. 201 means created, but if you just start typing `http.StatusCreated` it will either show up or autocomplete for you.